lib/ansible/modules/cloud/amazon/ec2_win_password.py

```diff
@@ -153,15 +158,12 @@ def main():
     else:
         try:
             with f:
-                key = RSA.importKey(f.read(), key_passphrase)
-        except (ValueError, IndexError, TypeError) as e:
+                key = load_pem_private_key(f.read(), key_passphrase, BACKEND)
+        except (ValueError, TypeError) as e:
             module.fail_json(msg = "unable to parse key file")
 
-    cipher = PKCS1_v1_5.new(key)
-    sentinel = 'password decryption failed!!!'
-
     try:
-        decrypted = cipher.decrypt(decoded, sentinel)
+        decrypted = key.decrypt(decoded, PKCS1v15())
     except ValueError as e:
         decrypted = None
 
```

Comment:
I'd be alright with making it cryptography-only, since I think folks using the cloud modules should be no newer systems, so they can get the new dep. It'd also be a nice carrot for "switch away from pycrypto, pretty please" for a certain segment of the userbase. 

Thoughts @s-hertel ?